### PR TITLE
Document the Target-Type LID keyword

### PR DESCRIPTION
**Pull request by cgay**
Document the Target-Type LID keyword, plus some minor improvements.

### CODE REVIEW

File: documentation/library-reference/source/lid.rst [hunk 1]
```diff
@@ -2,17 +2,15 @@
...
-LID solves this problem. It allows you to describe Dylan library sources
-in a form that any Dylan environment should be able to understand.
-Harlequin and other Dylan vendors have adopted LID to make it easier to
-port applications from one environment to another.
+The DRM defines an :drm:`interchange format for Dylan source files
+<Dylan_Interchange_Format>`, but does not define an interchange format for
+Dylan libraries.  Without such an agreed format, it would be difficult to
+import and build libraries developed using another Dylan implementation.
+
+Library Interchange Description (LID) format solves this problem. It allows you
```
Comment (housel) [inline documentation/library-reference/source/lid.rst:10]: **The** Library Interchange Description (LID) format...

File: documentation/library-reference/source/lid.rst [hunk 2]
```diff
@@ -2,17 +2,15 @@
...
+to describe Dylan library sources in a form that any Dylan environment should
+be able to understand.  Open Dylan adopted LID format to make it easier to port
```
Comment (housel) [inline documentation/library-reference/source/lid.rst:12]: the LID format

File: documentation/library-reference/source/lid.rst [hunk 3]
```diff
@@ -315,7 +313,7 @@ LID keyword
 Specifies the name of a LID file to process and includes the settings
 contained in that file into the current LID file.
 
-This is commonly used to share common definitions and settings between
+This is commonly used to share definitions and settings between
 platform or OS specific LID files.
```
Comment (housel) [inline documentation/library-reference/source/lid.rst:317]: OS-specific

File: documentation/library-reference/source/lid.rst [hunk 4]
```diff
@@ -344,6 +342,20 @@ An example JAM (for a library, not an executable) file might look like::
...
 within and return the output of that command.
 
+Target-Type:
+^^^^^^^^^^^^
+
+.. code-block:: dylan
+
+    Target-Type: *dll or executable*
+
+Specifies whether to generate a shared library (a.k.a, dynamic link library) or
```
Comment (housel) [inline documentation/library-reference/source/lid.rst:352]: dynamic-link library

File: documentation/library-reference/source/lid.rst [hunk 5]
```diff
@@ -344,6 +342,20 @@ An example JAM (for a library, not an executable) file might look like::
...
+an executable binary. Possible values are ``dll`` or ``executable``.
+
+If no target type is specified, Open Dylan builds an executable when the library
+is specified directly on the compiler command line or a shared library if the
```
Comment (housel) [inline documentation/library-reference/source/lid.rst:356]: command line, or

**Review by housel — APPROVED** [review]

**Comment by cgay** [comment]
All comments incorporated.

**Pull request merged**